azure-pipelines.yml

```diff
@@ -4,7 +4,7 @@ trigger:
 
 jobs:
 - job: Linux
-  timeoutInMinutes: 90
+  timeoutInMinutes: 150
```

Comment:
I'm not sure why the Azure jobs were taking so long before but they seem to have recovered as of the last push to dev. Can we revert these timeout increases?